sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/EventHubSharedKeyCredential.cs

```diff
@@ -22,14 +22,20 @@ internal sealed class EventHubSharedKeyCredential : TokenCredential
         ///   reported by the Azure portal.
         /// </summary>
         ///
-        private string SharedAccessKeyName { get; }
+        private string SharedAccessKeyName { get; set; }
```

Comment:
Looks like you absorbed the change already, but a quick note that we were asked to defer this for the next release in order to come up with a common design  across the languages and libraries so that the pattern is common.   It is definitely planned for public use in a near-term release.

Reply:
Thank you for the heads